metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg.py

```diff
@@ -134,15 +151,55 @@ def __init__(self, config: IcebergSourceConfig, ctx: PipelineContext) -> None:
         super().__init__(config, ctx)
         self.report: IcebergSourceReport = IcebergSourceReport()
         self.config: IcebergSourceConfig = config
+        self.ctx: PipelineContext = ctx
 
     @classmethod
     def create(cls, config_dict: Dict, ctx: PipelineContext) -> "IcebergSource":
         config = IcebergSourceConfig.parse_obj(config_dict)
         return cls(config, ctx)
 
     def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
+        # This source needs to overwrite standard `get_workunit_processor`, because it is unique in terms of usage
```

Comment:
I understand we are almost duplicating code from the parent but a couple of processors being removed, right?

In that case, could we just?

```
    def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
        # This source needs to overwrite standard `get_workunit_processor`, because it is unique in terms of usage
        # ...
        processors = super.get_workunit_processors()
        <filter out the two processors from processors>
        return processors
```